docs/l2-l3.rst

```diff
@@ -23,7 +23,10 @@ Create VLAN and Self IP
 4. After selecting the VLANS click :guilabel:`Manage Selected`.
 
    .. note::
-      - A VLAN configuration form is displayed to add VLANs and information for one or more self IPs. For VLAN, the fields to be configured are: (1) Tag (2) Interface. For self IPs, the fields to be configured are (1) Address (2) Netmask (3) Traffic-group (4) Port-lockdown.
+      - A VLAN configuration form is displayed to add VLANs and information for one or more self IPs. For VLAN, the fields to be configured are: (1) Tag (2) Interface (3) Partition (4) Route Domain. For self IPs, the fields to be configured are (1) Address (2) Netmask (3) Traffic-group (4) Port-lockdown.
+      - Partitions and Route Domains should be created from BIG-IP UI
+      - If Partition and Route Domain is not selected during creation of VLAN/Self IP, it will be assumed to be Common Partition and Route Domain 0 by default.
```

Comment:
If a Partition and Route Domain are not selected during the creation of the VLAN/self IP, it defaults to the Common Partition and Route Domain 0.